### PR TITLE
Enhanced Board parsing

### LINKED ISSUE #2: Allow the board to be initialised with simpler characters
At the moment the input to the board requires special line drawing characters.

Allow the creating of the board layout using basic ASCII characters.

Such as:

```
   +-----------+
   |           |
   +-----------+
```

### PR DESCRIPTION
**Pull request by smabuk**
This implements and closes #2 when finished

### CODE REVIEW

**Comment by smabuk** [comment]
Probably shouldn't have added the namespace additions into this PR but I was on a roll.

File: CapMan.Raylib/BoardRenderer.cs [hunk 1]
```diff
@@ -32,17 +33,19 @@ public void Render(Board board, int top, int left)
...
             Element.BottomRight => "bottom-right.png",
-            Element.Dot => "dot.png",
-            Element.PowerPill => "powerpill.png",
-            _ => throw new ArgumentException($"Invalid chess piece: {element}"),
-        };
+            Element.Dot         => "dot.png",
+            Element.PowerPill   => "powerpill.png",
+            Element.Corner      => "",
+            _ => throw new ArgumentException($"Invalid board element: {element}"),
+        });
```
Comment (jcollard) [inline CapMan.Raylib/BoardRenderer.cs:49]: Does your formatter automatically do this?

**Review by jcollard — APPROVED** [review]
Wonderful! Thank you.

My only concern is related to the "lined up" formatting you've added throughout:

```
            Element.Horizontal  => "horizontal.png",
            Element.Vertical    => "vertical.png",
            Element.TopLeft     => "top-left.png",
            Element.TopRight    => "top-right.png",
            Element.BottomLeft  => "bottom-left.png",
            Element.BottomRight => "bottom-right.png",
            Element.Dot         => "dot.png",
            Element.PowerPill   => "powerpill.png",
            Element.Corner      => "",
```

I understand the benefit / like it but, I'm not sure there is an easy way to provide this as a default formatter / enforce with a linter.

That said, greatly appreciate your efforts 😄

**Comment by smabuk** [comment]
I wish I could set up a linter to do this, but unfortunately not - I applied it in places I thought we wouldn't be likely to change often

**Pull request merged**